updating-bazel.md

```diff
@@ -6,8 +6,7 @@ title: Updating Bazel
 # Updating Bazel
```

Comment:
Where do you want to put this page?
How does it relate to https://docs.bazel.build/versions/master/backward-compatibility.html?

Reply:
It will be another link under https://docs.bazel.build/versions/master/backward-compatibility.html#how-to-follow-this-policy

https://docs.bazel.build/versions/master/backward-compatibility.html is the policy, this doc is a guidance for project maintainers